vowpalwabbit/parse_example_json.h

```diff
@@ -951,9 +951,9 @@ class DecisionServiceState : public BaseState<audit>
       else if (length == 11 && !_stricmp(str, "_labelIndex"))
         return &ctx.label_index_state;
     }
-    else if (length == 9 && !strncmp(str, "_deferred", 9))
+    else if (length == 10 && !strncmp(str, "_skipLearn", 10))
```

Comment:
Minor nit: is it possible to have these numbers be a constexpr constant based on the string you are feeding into strncmp?